### ISSUE TITLE
Get request to v1/schemas returns 502 error sometimes.

### ISSUE THREAD

**Issue by khushboo-rancher** [rancher/rancher#26727]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
1. Create a cluster(1 node all roles) and after it gets active immediately launch dashboard. 

**Result:**
Dashboard fails to launch as Get request to ```https://rancher-server/k8s/clusters/c-xxxxx/v1/schemas``` returns 502.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):v2.4.2
- Installation option (single install/HA):Single

Bug which will get resolved by this - https://github.com/rancher/dashboard/issues/393

**Comment by izaac** [comment]
This is a state issue either and is seen not only after cluster deployment. 

If I deploy one cluster with 3 nodes all roles and I shutdown one of those nodes after the cluster is available then go to the Nodes view of the Ember UI this node is shown unavailable `Kubelet stopped posting status.` and tainted.

Then trying to check this cluster Dashboard Overview returns the 502 fail-whale error page.

**Comment by izaac** [comment]
Closing this in favor of https://github.com/rancher/rancher/issues/28364 as the Dashboard has been heavily updated.
